### PR TITLE
OTJ: "committed a crime this turn" support

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/oko_the_ringleader.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Oko, the Ringleader
+ManaCost:2 G U
+Types:Legendary Planeswalker Oko
+Loyalty:3 
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigClone | TriggerDescription$ At the beginning of combat on your turn, CARDNAME becomes a copy of up to one target creature you control until end of turn, except he has hexproof.
+SVar:TrigClone:DB$ Clone | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select up to one target creature you control | TargetMin$ 0 | TargetMax$ 1 | Duration$ UntilEndOfTurn | PumpKeywords$ Hexproof
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/oko_the_ringleader.txt:6]: the KW isn't pumped but copiable

File: forge-gui/res/cardsfolder/upcoming/servant_of_the_stinger.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Servant of the Stinger
+ManaCost:1 B
+Types:Creature Human Warlock
+PT:1/3
+K:Deathtouch
+T:Mode$ DamageDone | ValidSource$ Card.Self | Execute$ TrigSearch | CombatDamage$ True | ValidTarget$ Player | CheckDefinedPlayer$ You.committedCrimeThisTurn | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, if you've committed a crime this turn, you may sacrifice Servant of the Stinger. If you do, search your library for a card, put it into your hand, then shuffle. (Targeting opponents, anything they control, and/or cards in their graveyards is a crime.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/servant_of_the_stinger.txt:6]: you could just do CheckSVar$ here to reuse the other syntax instead? 🤔

**Review by tool4ever — APPROVED** [review]

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/servant_of_the_stinger.txt (hunk 2 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/servant_of_the_stinger.txt:6]: Agh sorry I thought about it and decided to leave the PlayerProperty

I'd be shocked if we don't see a "each player who committed a crime this turn does X" especially in commander

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/servant_of_the_stinger.txt:6]: sure!

**Review by tool4ever — APPROVED** [review]

**Review by Hanmac — APPROVED** [review]

**Pull request merged**